test/dlc_tests/ec2/test_smdebug.py

```diff
@@ -2,17 +2,29 @@
 
 import pytest
 
-from test.test_utils import CONTAINER_TESTS_PREFIX, is_tf2
+from test.test_utils import CONTAINER_TESTS_PREFIX, is_tf2, is_tf1, get_dlc_images
 from test.test_utils.ec2 import get_ec2_instance_type
 
 
 SMDEBUG_SCRIPT = os.path.join(CONTAINER_TESTS_PREFIX, "testSmdebug")
 
 
-SMDEBUG_EC2_GPU_INSTANCE_TYPE = get_ec2_instance_type(default="p3.8xlarge", processor="gpu")
+SMDEBUG_EC2_GPU_INSTANCE_TYPE = get_ec2_instance_type(default="p2.8xlarge", processor="gpu")
```

Comment:
Are we changing this default instance type?

Reply:
No, i just wanted to test that this works as expected by setting the default instance type to p2.8x

Reply:
I will change back once confirmed